### PR TITLE
accounts/abi/bind: fix multi-value anonymous unmarshalling

### PR DESCRIPTION
**Pull request by karalabe**
This PR is a follow up to https://github.com/ethereum/go-ethereum/pull/2551, updating abigen too to use the correct unpack invocation.

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Thu May 12 13:30:17 UTC 2016_

**Comment by karalabe** [comment]
@obscuren Please review.

File: accounts/abi/abi_test.go [hunk 1]
```diff
@@ -305,10 +305,10 @@ func TestUnpackSetInterfaceSlice(t *testing.T) {
 		t.Fatal(err)
 	}
 	if *var1 != 1 {
-		t.Errorf("expected var1 to be 1, got", *var1)
+		t.Error("expected var1 to be 1, got", *var1)
```
Comment (obscuren) [inline accounts/abi/abi_test.go]: lol thanks

**Comment by obscuren** [comment]
👍 LGTM

**Pull request merged**